easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,20 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
         """Package software (e.g. into an RPM)."""
-        pass
+        rpmname = "HPCBIOS.20150211-%s-%s" % (self.name, self.version)
+        os.chdir(os.environ['TMPDIR'])
+        
+        
+        toolchaindep = "=".join([self.toolchain.name, self.toolchain.version])
+        if toolchaindep == "dummy=":
+            deplist = ""
+        else:
+            deplist = " ".join([" --dependency ", toolchaindep])
+        deplist += " ".join([" --dependency " + i 
+            for i in (map(lambda x:x["name"] + "=" + x["version"], self.cfg.dependencies()))
+        ])
+        cmd = "fpm --workdir $TMPDIR -t rpm --name %s -s dir %s -C %s" % (rpmname, deplist, self.installdir)
```

Comment:
the `rpm` must be set via variable (and later also from a config option).

you should also test if fpm is available (there's a which function somewhere in framework already taht can be used for this)